### ISSUE TITLE
Issues related to the releases/tags naming conventions

### ISSUE THREAD

**Issue by navarroaxel** [yarnpkg/yarn#1833]
**Do you want to request a *feature* or report a *bug*?** A possible bug.

**What is the current behavior?**
I think the ``v`` prefix on the tags name could cause an issues with the semantic versioning.

**If the current behavior is a bug, please provide the steps to reproduce.**
The issue is experienced here: winstonjs/winston-mongodb#77

**What is the expected behavior?**
The version should be compared using the semver value, and not using the release date.

**Please mention your node.js, yarn and operating system version.**
N/A

**Comment by kittens** [comment]
What does this mean? Can you please elaborate on the specific scenario you're referring to

**Comment by wyze** [comment]
I don't see any issue. If anything the author should learn how npm works when it comes to publishing. Just because the semver range is greater doesn't mean that is what will be installed when you do `yarn add winston-mongodb`.

```sh
$ yarn info winston-mongodb dist-tags
yarn info v0.17.0
{ latest: '1.6.5' }
✨  Done in 0.44s.
```

They published `2.0.0` to the `latest` tag and later published `1.6.5` to the `latest` tag thereby making it the "latest" version to install.
